float64[] joint_position
---
float64[] tcp_position
bool success
